clients/cli/src/main/java/org/apache/gravitino/cli/CommandActions.java

```diff
@@ -52,6 +54,8 @@ public class CommandActions {
     VALID_COMMANDS.add(PROPERTIES);
     VALID_COMMANDS.add(REVOKE);
     VALID_COMMANDS.add(GRANT);
+    VALID_COMMANDS.add(ENABLE);
+    VALID_COMMANDS.add(DISABLE);
```

Comment:
I don't think we should add "enable/disable" as top-level commands considering that these actions are only applicable to metalakes and catalogs. I'd model these as toggling a flag on metalakes and catalogs. i.e.

```shell
gcli metalake update --enable
```

or

```
gcli metalake enable
```

In that spirit, I'd name the classes/methods as `MetalakeEnable` rather than `EnableMetalake`, for example.

Reply:
![image](https://github.com/user-attachments/assets/57d84627-3450-4e1f-94f2-5e389e36bc2a)

If we not added these commands to VALID_COMMANDS, we would have needed to modify the command validation flow.
I will change the name, thank you

Reply:
If added as option, isn't that a bit much for the client.

Reply:
> If added as option, isn't that a bit much for the client.

It is fine, IMHO. These are not frequent actions a user would perform every now and then.

Reply:
Let me change the code，thank you.

Reply:
Hi @justinmclean @tengqm , I’ve finished updating the code. Please take a look at the PR again when you have time.